assets/js/src/Popover.js

```diff
@@ -13,6 +13,10 @@ Kanboard.Popover.prototype.listen = function() {
         self.close(e);
     });
 
+    $(document).on("click", "#popover-container", function(e) {
+        self.close(e);
```

Comment:
I already built something to address this topic @ https://github.com/kanboard/kanboard/issues/2847